lib/react/rails/railtie.rb

```diff
@@ -46,17 +46,17 @@ class Railtie < ::Rails::Railtie
                      tmp_path.join('react.js'))
         FileUtils.cp(::React::Source.bundled_path_for('JSXTransformer.js'),
                      tmp_path.join('JSXTransformer.js'))
-        app.assets.prepend_path tmp_path
+        app.config.assets.paths.unshift(tmp_path)
```

Comment:
@josh I couldn't use `config.assets.configure` here because it's `nil` and it's only set in `after_initialize`: https://github.com/rails/sprockets-rails/blob/master/lib/sprockets/railtie.rb#L142

May I use operate directly on `app.config.assets.paths` here?